homeassistant/components/overkiz/climate_entities/hitachi_air_to_air_heat_pump_ovp.py

```diff
@@ -0,0 +1,359 @@
+"""Support for HitachiAirToAirHeatPump."""
+from __future__ import annotations
+
+from typing import Any, cast
+
+from pyoverkiz.enums import OverkizCommand, OverkizCommandParam, OverkizState
+
+from homeassistant.components.climate import (
+    FAN_AUTO,
+    FAN_HIGH,
+    FAN_LOW,
+    FAN_MEDIUM,
+    PRESET_NONE,
+    SWING_BOTH,
+    SWING_HORIZONTAL,
+    SWING_OFF,
+    SWING_VERTICAL,
+    ClimateEntity,
+    ClimateEntityFeature,
+    HVACMode,
+)
+from homeassistant.const import ATTR_TEMPERATURE, UnitOfTemperature
+
+from ..const import DOMAIN
+from ..coordinator import OverkizDataUpdateCoordinator
+from ..entity import OverkizEntity
+
+PRESET_HOLIDAY_MODE = "holiday_mode"
+FAN_SILENT = OverkizCommandParam.SILENT
+AUTO_MANU_MODE_STATE = OverkizState.CORE_AUTO_MANU_MODE
```

Comment:
My preference would be to not add a lot of constants for these parameters. These enums are not used a lot, and are very readable already. Introducing another constant to just point to the ENUM makes it not very readable in my opinion.

Would you mind removing all these extra variables? The ones for temperature you can keep.

Reply:
Makes sense. I would like to keep `FAN_SILENT`, though, because it is missing in the Overkiz library enum, and I want the code to look consistent where we use the `FAN_HIGH`, `FAN_LOW`... etc.

Reply:
Best is to set FAN_SILENT to a string. It is not directly coupled to the Overkiz command name.